Tensile/KernelWriterAssembly.py

```diff
@@ -5396,13 +5397,13 @@ def calculateLoopNumIter(self, kernel, loopIdx, isPap):
         kStr += inst("s_cmov_b32", loopCounter, hex(0), "numIter=0 if gsuSimIdx!=remainder")
 
       # if tail numIter == 0 skip altogether
-      tailLoopLabelEnd = self.getNamedLabel("TailLoopEnd%s"%(loopChar) )
+      skipTailLoopLabel = self.getNamedLabel("SkipTailLoop%s"%(loopChar) )
       kStr += inst("s_cmp_eq_u32", loopCounter, \
           hex(0), "numIter%s == 0"%loopChar )
       kStr += inst("s_mov_b32", sgpr("OrigLoopCounter"), 0, \
           "repurpose to count each localRead increment")
       kStr += inst("s_cbranch_scc1 %s"\
-          % tailLoopLabelEnd, \
+          % skipTailLoopLabel, \
```

Comment:
Use a SkipTailLoop instead of TailLoopEnd, since TailLoopEnd now is scattered in several tail-loops when DULDS>1. Using this "Skip" to enclose the whole tail loop should be easier to read.